### DISCUSSION TITLE
unexpected token o in JSON at position 1

### DISCUSSION THREAD

**Discussion by brypie** [louislam/dockge#283] [Ask for Help] [answered]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/dockge/security/policy)

### 📝 Describe your problem

Hi, running Dockge version 1.3.0 at the moment, and I can see there are some updates available for some of my containers.
However, when I try the update facility within dockge, I keep getting the following error shown : 
"unexpected token o in JSON at position 1"  and the container fails to update.

Is this a known issue?
Any way to get around it?

(Also happens when I try to update dockge!)

TIA

### 📝 Error Message(s) or Log

unexpected token o in JSON at position 1 

### 🐻 Dockge Version

1.3.0

### 💻 Operating System and Arch

Ubuntu 22.04 Server x64

### 🌐 Browser

Chrome

**Comment by louislam** [comment, marked as answer]
Update to 1.3.3

**Reply by brypie** [reply]
Ok fair enough - what's the best way to do that seeing as I can't do it from dockge itself?

**Reply by brypie** [reply]
Never mind!  Stopped and removed the container, then ran compose up command again which pulled the latest image.

Thanks
